src/erlcloud_ecs.erl

```diff
@@ -1177,7 +1178,8 @@ decode_container_overrides_list(V, Opts) ->
 %%%------------------------------------------------------------------------------
 %% CreateCluster
 %%%------------------------------------------------------------------------------
--type create_cluster_opt() :: {cluster_name, string_param()}.
+-type create_cluster_opt() :: {cluster_name, string_param()} |
+                              {out, record}.
```

Comment:
I don't know if this is a proper option, or not, but it's referenced in the tests in a lot of different places.

Reply:
looks like there is a type, `out_opt()`, that should be in all these.

Reply:
Fixed in 18a08be. I think I might have fixed previous occurrences also, but since I was touching it... (I guess no harm done)